docker/observability/README.md

```diff
@@ -0,0 +1,50 @@
+# Observability stack
+
+The config files in this directory go together with `docker-compose.observability.yaml` and provide
+a reference for a potential observability stack.
+
+**Disclaimer**
+
+The chosen vendors/technologies here merely serve as an example - it's simply a stack we're somewhat
+comfortable with. Open Forms itself is vendor agnostic and the principles demonstrated apply to
+competing vendors too.
+
+## Bringing up the services
+
+From the root of the repository:
+
+```bash
+docker compose up -d
```

Comment:
Do we need that?What is the result of it?I would go only for the one below as we do with all the other containers that's why I am asking.

Reply:
this actually brings up Open Forms itself, from which we are scraping the logs to display in Grafana.

So the second docker compose brings up Loki, Promtail + Grafana, the first one ensures that there's actually something visible in them :)